src/components/organisms/AppRouter/Provided.tsx

```diff
@@ -1,48 +1,26 @@
 import React from "react";
 
 import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
-import {
-  RelatedVenuesProvider,
-  RelatedVenuesProviderProps,
-} from "hooks/useRelatedVenues";
-import { WorldUsersProvider, WorldUsersProviderProps } from "hooks/users";
+import { RelatedVenuesProvider } from "hooks/useRelatedVenues";
 import { useVenueId } from "hooks/useVenueId";
 
-export type EmptyProviderProps = Partial<
-  RelatedVenuesProviderProps & WorldUsersProviderProps
->;
-
-const EmptyProvider: React.FC<EmptyProviderProps> = ({ children }) => {
-  return <>{children}</>;
-};
-
 export interface ProvidedProps {
   withRelatedVenues?: boolean;
-  withWorldUsers?: boolean;
 }
 
 export const Provided: React.FC<ProvidedProps> = ({
   children,
   withRelatedVenues = false,
-  withWorldUsers = false,
 }) => {
   const venueId = useVenueId();
 
   const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
 
-  const MaybeRelatedVenuesProvider: React.FC<RelatedVenuesProviderProps> = withRelatedVenues
-    ? RelatedVenuesProvider
-    : EmptyProvider;
-
-  const MaybeWorldUsersProvider: React.FC<WorldUsersProviderProps> = withWorldUsers
-    ? WorldUsersProvider
-    : EmptyProvider;
+  if (!withRelatedVenues) return <>{children}</>;
```

Comment:
[NB] You can just `return children`

Reply:
I don't think I can. I tried, but 
![image](https://user-images.githubusercontent.com/63194656/134667851-65de5918-1961-44e8-9b33-32495823c258.png)